docs/PORTABLE_SERVICES.md

```diff
@@ -123,33 +121,32 @@ This command does the following:
 
 4. For each such unit a "profile" drop-in is linked in. This "profile" drop-in
    generally contains security options that lock down the service. By default
-   the `default` profile is used, which provides a medium level of
-   security. There's also `trusted` which runs the service at the highest
-   privileges, i.e. host's root and everything. The `strict` profile comes with
-   the toughest security restrictions. Finally, `nonetwork` is like `default`
-   but without network access. Users may define their own profiles too (or
-   modify the existing ones)
+   the `default` profile is used, which provides a medium level of security.
+   There's also `trusted`, which runs the service with no restrictions, i.e. in
+   the host file system root and with full privileges. The `strict` profile
+   comes with the toughest security restrictions. Finally, `nonetwork` is like
+   `default` but without network access. Users may define their own profiles
+   too (or modify the existing ones).
 
 And that's already it.
 
 Note that the images need to stay around (and in the same location) as long as the
 portable service is attached. If an image is moved, the `RootImage=` line
-written to the unit drop-in would point to an non-existing place, and break the
-logic.
+written to the unit drop-in would point to an non-existent path, and break
+access to the image.
 
 The `portablectl detach` command executes the reverse operation: it looks for
-the drop-ins and the unit files associated with the image, and removes them
-again.
+the drop-ins and the unit files associated with the image, and removes them.
 
 Note that `portablectl attach` won't enable or start any of the units it copies
 out by default, but `--enable` and `--now` parameter are available as shortcuts.
 The same is true for the opposite `detach` operation.
 
-A `portablectl reattach` command is made available to combine a `detach` with an
-`attach`, and it is useful in case an image gets upgraded, as it allows a to
-perform a `restart` operation on the unit(s) instead of `stop` plus `start`,
-thus providing lower downtime and avoiding losing runtime state associated with
-the unit such as the file descriptor store.
+The `portablectl reattach` command combines a `detach` with an `attach`. It is
+useful in case an image gets upgraded, as it performs a `restart` operation on
```

Comment:
The detach itself does not do a restart (unless --now is used), that's why I used the "allows" term

Reply:
Fixed.